docs/articles/new-expensify/reports-and-expenses/Create-and-Submit-Reports.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Create-and-Submit-Reports.md
+description: Learn how to use New Expensify’s report-first flow to create, edit, submit, and retract expense reports.
+keywords: [New Expensify, create report, submit report, retract report, undo submit, undo close, add expenses, fix report]
+---
+<div id="new-expensify" markdown="1">
+
+Easily manage your business expenses in New Expensify with our streamlined report-first workflow. This guide walks you through creating, editing, submitting, and even retracting expense reports when needed.
+
+# Create a Report
+
+Follow these steps to build and submit an expense report manually.
+
+## Step 1: Create an empty report
+
+1. Tap the green **➕ Create** button at the bottom of your screen.
+2. Select **Create report**.
+3. Choose the workspace for your report:
+   - If you're in one workspace, it will be selected automatically.
```

Comment:
@RachCHopkins can we revert point #3 here? It's not accurate. The only time a workspace selector shows up is when the user has set their personal policy as the default policy accidentally via OldDot AND they have two or more group workspaces, thus we can't guess. 

I think it's bette to kill this point entirely.